src/Resources/views/CRUD/base_list.html.twig

```diff
@@ -186,7 +186,7 @@ file that was distributed with this source code.
                                             <label class="checkbox" for="{{ admin.uniqid }}_all_elements">
                                                 <input type="checkbox" name="all_elements" id="{{ admin.uniqid }}_all_elements">
                                                 {{ 'all_elements'|trans({}, 'SonataAdminBundle') }}
-                                                ({{ admin.datagrid.pager.countResults() }})
+                                                ({{ admin.datagrid.pager.countResults() }}{% if not admin.datagrid.pager.deterministic %}+{% endif %})
```

Comment:
Since the method is not mandatory, we might need an internal twig extension
```
public function isDeterministic(PagerInterface $pager): bool
{
     if (!method_exist($pager, 'isDeterministic')) {
         return true;
     }

     return $pager->isDeterministic();
}
```